great_expectations/data_context/types/base.py

```diff
@@ -653,6 +654,8 @@ def __init__(
             self.azure_options = azure_options
         if gcs_options is not None:
             self.gcs_options = gcs_options
+        if credentials_info is not None:
+            self.credentials_info = credentials_info
```

Comment:
Is there a particular reason for using `credentials_info`? If the field is specific to BigQuery, could we perhaps change it to something like `bigquery_credentials`?

We're aiming to split up these large schemas to better match the class hierarchy we have in the codebase so maybe `credentials` will do (but not currently due to a namespace collision).

Reply:
Yeah this is a named parameter to the BigQuery SQLAlchemy Dialect, which accepts a dict of credentials. The driver also accepts 'credentials_path', which is why broadening it to 'bigquery_credentials' may not be specific enough.

Reply:
this is good to know, thank you! going to close this comment